e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -596,6 +626,15 @@
           "timeToLiveSecs": 86400
         }
       }
+    },
+    "metricsCollector": {
+      "properties.desired": {
+        "upstreamProtocol": "<UpstreamProtocol>",
+        "offlineFrequency": "<NetworkController.OfflineFrequency0>",
+        "onlineFrequency": "<NetworkController.OnlineFrequency0>",
+        "runsCount": "<NetworkController.RunsCount0>",
+        "testDuration": "<TestDuration>"
```

Comment:
You may need to consider testStartDelay and plus some buffer for metrics collector module.

Reply:
Do we want to delay the metrics collector? It is not part of the test, so I was thinking I could start immediately.

Reply:
since TestDuration is used as additional metrics tag, that's fine.  I will update connectivity test to shut down iotedge service after test.